packages/fxa-settings/src/components/Icons/en.ftl

```diff
@@ -0,0 +1,43 @@
+
+## Images - these are all aria labels used for illustrations
+## Aria labels are used as alternate text that can be read aloud by screen readers.
+
+alert-icon-aria-label =
+    .aria-label = Alert
+
+authenticator-app-aria-label =
+    .aria-label = Authenticator Application
+
+backup-codes-icon-aria-label =
+    .aria-label = Backup codes enabled
+
+backup-codes-disabled-icon-aria-label =
+    .aria-label = Backup codes disabled
+
+# An icon of phone with text message. A back recovery phone number
+backup-recovery-sms-icon-aria-label =
+    .aria-label = Recovery SMS enabled
+
+# Disabled version of backup-recovery-sms-icon-aria-label
+backup-recovery-sms-disabled-icon-aria-label =
+    .aria-label = Recovery SMS disabled
+
+# used indicate a success, as in checked!
+checkmark-icon-aria-label =
+    .aria-label = Check
+
+# Used to decorate a code you enter for verification purposes
+code-icon-aria-label =
+    .aria-label = Code
+
+# Used to select Canada as country code for phone number
+canadian-flag-icon-aria-label =
+    .aria-label = Canadian Flag
+
+# Used to select United States as a country code for phone number
+usa-flag-icon-aria-label =
+    .aria-label = USA Flag
```

Comment:
I'd recommend `United States` here to be consistent with what we use for the region name in this and other products.